op-deployer/pkg/deployer/upgrade/flags.go

```diff
@@ -0,0 +1,34 @@
+package upgrade
+
+import (
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer"
+	v200 "github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/upgrade/v200"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	ConfigFlag = &cli.StringFlag{
+		Name:  "config",
+		Usage: "path to the config file",
+	}
+	OverrideArtifactsURLFlag = &cli.StringFlag{
+		Name:  "override-artifacts-url",
+		Usage: "override the artifacts URL",
+	}
+)
+
+var Commands = cli.Commands{
+	&cli.Command{
+		Name:  "v200",
```

Comment:
Also wonder if we could change this from `v200` -> `v2_0_0` to preserve the semver better

Reply:
I can make the invocation `v2.0.0`.